projects/habitat_ovmm/configs/agent/floorplanner_eval.yaml

```diff
@@ -36,6 +36,10 @@ AGENT:
     explored_radius: 150       # radius (in centimeters) of visually explored region
     must_explore_close: False
 
+  SKILLS:
+    GAZE:
+      CHECKPOINT_PATH: data/checkpoints/gaze.pth
```

Comment:
should this be under the projects folder? or should we put all project configs under data, so that we can be consistent?

Reply:
I verified that the entire data/ can now be built and placed under projects/habitat_ovmm. We will need to run `python eval_vectorized.py` from inside the `projects/habitat_ovmm`. That all project configs, data and checkpoints will be under `projects/habitat_ovmm`. The only issue is that the real world experiments need the checkpoints and clip embeddings.

Reply:
resolved - we will keep it where it is but checkpoints should be under ovmm